src/interface/ChangelogPanel/ChangelogPanel.tsx

```diff
@@ -26,7 +27,7 @@ const ChangelogPanel = () => {
           onChange={(e) => setChangelogType(Number(e.target.value))}
         >
           <option value={0}>
-            <Trans id="interface.changelogPanel.option.core">Core</Trans>
+            {i18n._('interface.changelogPanel.option.core')}
```

Comment:
Since Lingui v3 i18n._ has been replaced with the `t` macro. It is used all over the codebase for an example.

Reply:
Ah, yeah I have seen that all over, didn't realize it was the same thing. Updated!